src/Renderer/ThreeExtended/Feature2Mesh.js

```diff
@@ -399,6 +453,7 @@ export default {
      * @param {number|function} options.altitude - define the base altitude of the mesh
      * @param {number|function} options.extrude - if defined, polygons will be extruded by the specified amount
      * @param {object|function} options.color - define per feature color
+     * @param {function} options.batchId - optional function to create batchId attribute. It is passed the feature property and the feature index
```

Comment:
you could add` [] `for optional parameter : `[options.batchId]`

Reply:
done